main.css

```diff
@@ -9,77 +9,131 @@ h1 {
 
 
 form {
-/* Just to center the form on the page */
-margin: 0 auto;
-width: 400px;
-/* To see the outline of the form */
-padding: 1em;
-border: 3px solid aqua;
-border-radius: 1em;
+    /* Just to center the form on the page */
+    margin: 0 auto;
+    width: 500px;
+    /* To see the outline of the form */
+    padding: 1em;
+    border: 3px solid aqua;
+    border-radius: 1em;
+
 
+}
 
+label {
+    /* labels display on top of field. */
+    display: block;
 }
 
 .print {
-margin:  auto;
-width: 400px;
-padding: 1em;
-border-radius: 1em;
-background-color: aquamarine;
+    /* this is the div where user input prints after submit */
+    margin: auto;
+    width: 500px;
+    padding: 1em;
+    border-radius: 1em;
+    background-color: aquamarine;
 }
 
 
-form div + div {
-margin-top: 1em;
+.wrapper {
+    /* this is the outer div */
+    margin-top: 1em;
 }
 
-label {
-/* To make sure that all labels have the same size and are properly aligned */
-display: inline-block;
-width: 90px;
-text-align: right;
+.container_user_name {
```

Comment:
all the `container_[something]` classes apply the same styles, so they can all be the same class. give this class a more generic name that describes its purpose and use it for everything that currently has its own `container_[something]` class